test/IECoreHoudini/LiveSceneTest.py

```diff
@@ -1056,8 +1056,10 @@ def testBrokenSop( self ) :
 
 		# forcing a cook error
 		hou.parm('/obj/box1/actualBox/sizex').setExpression( "fake" )
+
+		box1 = scene.scene( [ "sub1", "box1" ] )
```

Comment:
Why is it necessary to re-acquire the box1 variable here?

Reply:
Doesn't need to be re-acquired. Fixed in 909ea36